### ISSUE TITLE
textarea_fp is exploitable

### ISSUE THREAD

**Issue by dmitris** [yahoo/webseclab#17]
The following fires an alert:

```
http://10.89.11.247:8080/xss/reflect/textarea1_fp?in=%3C%2Ftextarea%3E%3Cscript%3Ealert%28/XSS/%29%3B%3C%2Fscript%3E%3Ctextarea%3E
```

**Comment by dmitris** [comment]
should simplify the regexp handling with a simpler string processing function

**Comment by dmitris** [comment]
Fixed by #18
